[0m[[0minfo[0m] [0mRunning FooParChecks -fpppn 8 -fpnp 8[0m
[0m[[0mdebug[0m] [0mWaiting for threads to exit or System.exit to be called.[0m
[0m[[0mdebug[0m] [0m  Classpath:[0m
[0m[[0mdebug[0m] [0m	/home/felix/git/FooPar/target/scala-2.11/classes[0m
[0m[[0mdebug[0m] [0m	/home/felix/.ivy2/cache/org.scala-lang/scala-library/jars/scala-library-2.11.1.jar[0m
[0m[[0mdebug[0m] [0m	/home/felix/.ivy2/cache/com.typesafe.akka/akka-actor_2.11/jars/akka-actor_2.11-2.3.4.jar[0m
[0m[[0mdebug[0m] [0m	/home/felix/.ivy2/cache/com.typesafe/config/bundles/config-1.2.1.jar[0m
[0m[[0mdebug[0m] [0m	/home/felix/.ivy2/cache/com.typesafe.akka/akka-remote_2.11/jars/akka-remote_2.11-2.3.4.jar[0m
[0m[[0mdebug[0m] [0m	/home/felix/.ivy2/cache/io.netty/netty/bundles/netty-3.8.0.Final.jar[0m
[0m[[0mdebug[0m] [0m	/home/felix/.ivy2/cache/com.google.protobuf/protobuf-java/bundles/protobuf-java-2.5.0.jar[0m
[0m[[0mdebug[0m] [0m	/home/felix/.ivy2/cache/org.uncommons.maths/uncommons-maths/jars/uncommons-maths-1.2.2a.jar[0m
[0m[[0mdebug[0m] [0mWaiting for thread run-main-3 to terminate.[0m
[0m[[31merror[0m] [0m(run-main-3) java.lang.ClassNotFoundException: FooParChecks[0m
java.lang.ClassNotFoundException: FooParChecks
	at java.lang.ClassLoader.findClass(ClassLoader.java:531)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
	at sbt.classpath.ClasspathFilter.loadClass(ClassLoaders.scala:53)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:270)
	at sbt.Run.getMainMethod(Run.scala:77)
	at sbt.Run.run0(Run.scala:64)
	at sbt.Run.sbt$Run$$execute$1(Run.scala:54)
	at sbt.Run$$anonfun$run$1.apply$mcV$sp(Run.scala:58)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:58)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:58)
	at sbt.Logger$$anon$4.apply(Logger.scala:90)
	at sbt.TrapExit$App.run(TrapExit.scala:244)
	at java.lang.Thread.run(Thread.java:744)
[0m[[0mdebug[0m] [0m	Thread run-main-3 exited.[0m
[0m[[0mdebug[0m] [0mInterrupting remaining threads (should be all daemons).[0m
[0m[[0mdebug[0m] [0mSandboxed run complete..[0m
java.lang.RuntimeException: Nonzero exit code: 1
	at scala.sys.package$.error(package.scala:27)
	at sbt.BuildCommon$$anonfun$toError$1.apply(Defaults.scala:1708)
	at sbt.BuildCommon$$anonfun$toError$1.apply(Defaults.scala:1708)
	at scala.Option.foreach(Option.scala:236)
	at sbt.BuildCommon$class.toError(Defaults.scala:1708)
	at sbt.Defaults$.toError(Defaults.scala:35)
	at sbt.Defaults$$anonfun$runMainTask$1$$anonfun$apply$34$$anonfun$apply$35.apply(Defaults.scala:678)
	at sbt.Defaults$$anonfun$runMainTask$1$$anonfun$apply$34$$anonfun$apply$35.apply(Defaults.scala:676)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:42)
	at sbt.std.Transform$$anon$4.work(System.scala:64)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18)
	at sbt.Execute.work(Execute.scala:244)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[0m[[31merror[0m] [0m(compile:[31mrunMain[0m) Nonzero exit code: 1[0m
